lib/services/internalEngine.js

```diff
@@ -43,8 +42,8 @@ function InternalEngine (kuzzle) {
   this.init = function () {
     if (!this.client) {
       this.client = new Elasticsearch.Client({
-        host: kuzzle.config[this.engineType].hosts,
-        apiVersion: kuzzle.config[this.engineType].apiVersion
+        host: kuzzle.config.services.db.host + ':' + kuzzle.config.services.db.port,
```

Comment:
the thing is that despite the fact the internalEngine source is located in the services directory, it is actually not initialized by the service layer but directly from Kuzzle start process.

I agree this would make the service look more consistent but the sole purpose of passing the service configuration from `lib/services/index.js` is to deal with mutliple backend/aliases cases where it could be tricky to get the proper config from the constructor.

In this specific case, this logic is not needed.